indykite/common_schema.go

```diff
@@ -57,6 +59,29 @@ func regionSchema() *schema.Schema {
 	}
 }
 
+func ikgSizeSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: `IKG size that will be allocated, which correspond also to amount of CPU nodes (default 2GB).`,
+		ValidateFunc: validation.StringInSlice([]string{
+			"2GB", "4GB", "8GB", "16GB", "32GB", "64GB", "128GB", "192GB", "256GB", "384GB", "512GB",
+		}, false),
+	}
+}
+
+func replicaRegionSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeString,
+		Optional: true,
+		Description: `Replica region specify where the second IKG is created.
```

Comment:
grammar: "Replica region **specifies** where..."
I would also say "replica IKG" instead of "second IKG"